src/components/partnership/partnership.service.ts

```diff
@@ -491,6 +513,12 @@ export class PartnershipService {
   }
 
   async update(input: UpdatePartnership, session: ISession) {
+    if (input.fundingType && !this.canAddFundingType(input.types)) {
```

Comment:
This needs to account for existing types if it's not passed in.

Reply:
Aha got it.